packages/discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -308,6 +308,15 @@ def populate_track_record_metadata(track_record: Track, track_metadata, handle,
                     continue
                 setattr(track_record, key, track_metadata[key])
 
+    # Default release_date to created_at or current time if not set
```

Comment:
i think created_at should always be there. it might be simpler to just add release_date to here?
https://github.com/AudiusProject/audius-protocol/blob/4bf1f8be5bc651b1e39cf9b7d76257499a2e8de8/packages/discovery-provider/src/tasks/entity_manager/entities/track.py#L413-L431

Reply:
makes sense, i'll re-test and make sure it doesn't run into any issues